relay-server/src/statsd.rs

```diff
@@ -577,9 +576,12 @@ impl TimerMetric for RelayTimers {
             RelayTimers::ReplayRecordingProcessing => "replay.recording.process",
             RelayTimers::GlobalConfigRequestDuration => "global_config.requests.duration",
             RelayTimers::ProcessMessageDuration => "processor.message.duration",
-            RelayTimers::ProjectCacheMessageDuration => "project_cache.message.duration",
             RelayTimers::BufferMessageProcessDuration => "buffer.message.duration",
             RelayTimers::ProjectCacheTaskDuration => "project_cache.task.duration",
+            RelayTimers::LegacyProjectCacheMessageDuration => {
+                "legacy_project_cache.message.duration"
+            }
+            RelayTimers::LegacyProjectCacheTaskDuration => "legacy_project_cache.task.duration",
```

Comment:
Do you actually want to rename the string ids of the metrics? This makes querying both old and new a bit more annoying.

Reply:
Yes, at first I wanted to add new metrics for the new project cache, but ended up deciding to rather treat the 'old' one as a new legacy layer and treat the new project cache as a replacement which it is. Once spool v1 is gone, removing the legacy service is then just deleting metrics and code.